### PR TITLE
examples: refactor a bit the examples

### PR DESCRIPTION
**Pull request by antoniivanov**
- moved the common storage code to vdk-storage library
- refactored confluence-reader to directly use it instead of using file

### CODE REVIEW

File: examples/confluence-reader/fetch_confluence_space.py [hunk 1]
```diff
@@ -1,76 +1,33 @@
...
-                doc["metadata"], doc["data"], doc["metadata"].get("deleted", False)
-            )
-            for doc in existing_docs
-        ]
-
-    existing_docs_dict = {doc.metadata["id"]: doc for doc in existing_docs}
-
-    for doc in new_docs:
-        existing_docs_dict[doc.metadata["id"]] = doc
+def merge_docs(existing_docs, new_docs) -> list:
```
Comment (duyguHsnHsn) [inline examples/confluence-reader/fetch_confluence_space.py:15]: Please, change the README accordingly, too

File: examples/confluence-reader/fetch_confluence_space.py [hunk 2]
```diff
@@ -170,34 +124,31 @@ def run(job_input: IJobInput):
...
-        data_file,
+        all_docs,
         confluence_reader.fetch_all_pages_in_confluence_space(parent_page_id),
     )
 
     # TODO: it would be better to save each page in separate row.
     # But that's quick solution for now to pass the data to the next job
 
-    storage.store(storage_name, pathlib.Path(data_file).read_text())
+    storage.store(storage_name, [doc.serialize() for doc in all_docs])
```
Comment (yonitoo) [inline examples/confluence-reader/fetch_confluence_space.py:154]: What is the intention here behind the docs serialization? maybe add a comment

**Review by yonitoo — COMMENTED** [review]
Looks good to me. Left 1 comment.

File: examples/confluence-reader/fetch_confluence_space.py (hunk 2 above)
Comment (antoniivanov) [reply examples/confluence-reader/fetch_confluence_space.py:156]: It's not a new code.

File: examples/pgvector-embedder/01_local_properties.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    props = job_input.get_all_properties()
+    props[key] = value
+    job_input.set_all_properties(props)
+
+
+def run(job_input: IJobInput):
+    set_property(
+        job_input,
+        "storage_connection_string",
+        "postgresql://pgadmin:ePgqsEQ2823S8J7L2w4gVziNz7IWm0@10.148.186.71:5432/spulov-test",
```
Comment (duyguHsnHsn) [inline examples/pgvector-embedder/01_local_properties.py:21]: remove this

**Review by duyguHsnHsn — APPROVED** [review]

File: examples/pgvector-embedder/01_local_properties.py (hunk 3 above)
Comment (antoniivanov) [reply examples/pgvector-embedder/01_local_properties.py:21]: Thanks!

**Review by yonitoo — APPROVED** [review]

**Pull request merged**